diff --git a/node_modules/@railgun-community/engine/dist/contracts/railgun-smart-wallet/railgun-smart-wallet.js b/node_modules/@railgun-community/engine/dist/contracts/railgun-smart-wallet/railgun-smart-wallet.js
index b942514..db2df20 100644
--- a/node_modules/@railgun-community/engine/dist/contracts/railgun-smart-wallet/railgun-smart-wallet.js
+++ b/node_modules/@railgun-community/engine/dist/contracts/railgun-smart-wallet/railgun-smart-wallet.js
@@ -241,6 +241,10 @@ class RailgunSmartWalletContract extends events_1.default {
      * @param latestBlock - block to scan to
      */
     async getHistoricalEvents(chain, initialStartBlock, latestBlock, getNextStartBlockFromValidMerkletree, eventsListener, eventsNullifierListener, eventsUnshieldListener, setLastSyncedBlock) {
+
+        // Cookbook does not need event scanning.
+        return;
+
         const engineV3StartBlockNumber = RailgunSmartWalletContract.getEngineV3StartBlockNumber(chain);
         const engineV3ShieldEventUpdate030923BlockNumber = RailgunSmartWalletContract.getEngineV3ShieldEventUpdate030923BlockNumber(chain);
         // TODO: Possible data integrity issue in using commitment block numbers.
