diff --git a/node_modules/@railgun-community/engine/dist/contracts/railgun-smart-wallet/railgun-smart-wallet.js b/node_modules/@railgun-community/engine/dist/contracts/railgun-smart-wallet/railgun-smart-wallet.js
index c3fab66..6271988 100644
--- a/node_modules/@railgun-community/engine/dist/contracts/railgun-smart-wallet/railgun-smart-wallet.js
+++ b/node_modules/@railgun-community/engine/dist/contracts/railgun-smart-wallet/railgun-smart-wallet.js
@@ -183,6 +183,9 @@ class RailgunSmartWalletContract extends events_1.default {
         // If we miss an unshield/nullifier for some reason, we won't pick it up .
         // For missed unshields, this will affect the way transaction history is displayed (no balance impact).
         // For missed nullifiers, this will incorrectly show a balance for a spent note.
+
+        return;
+
         let currentStartBlock = initialStartBlock;
         // Current live events - post v3 update
         const eventFilterNullified = this.contract.filters.Nullified();
