diff --git a/node_modules/ethers/lib.commonjs/providers/subscriber-polling.js b/node_modules/ethers/lib.commonjs/providers/subscriber-polling.js
index 2dce1e0..b2ce8b6 100644
--- a/node_modules/ethers/lib.commonjs/providers/subscriber-polling.js
+++ b/node_modules/ethers/lib.commonjs/providers/subscriber-polling.js
@@ -222,7 +222,7 @@ class PollingEventSubscriber {
             return;
         }
         const filter = copy(this.#filter);
-        filter.fromBlock = this.#blockNumber + 1;
+        filter.fromBlock = this.#blockNumber; // Updated to fix a bug in polling scans for the same filtered events across consecutive blocks (ie. during automated tests).
         filter.toBlock = blockNumber;
         const logs = await this.#provider.getLogs(filter);
         // No logs could just mean the node has not indexed them yet,
