diff --git a/node_modules/ethers/lib.commonjs/providers/subscriber-polling.js b/node_modules/ethers/lib.commonjs/providers/subscriber-polling.js
index 162d6c8..6873c3c 100644
--- a/node_modules/ethers/lib.commonjs/providers/subscriber-polling.js
+++ b/node_modules/ethers/lib.commonjs/providers/subscriber-polling.js
@@ -236,6 +236,7 @@ class PollingEventSubscriber {
         }
         const filter = copy(this.#filter);
         filter.fromBlock = this.#blockNumber + 1;
+        filter.fromBlock = this.#blockNumber; // Updated to fix a bug in polling scans for the same filtered events across consecutive blocks (ie. during automated tests).
         filter.toBlock = blockNumber;
         const logs = await this.#provider.getLogs(filter);
         // Undo debounce state
diff --git a/node_modules/ethers/lib.esm/providers/subscriber-polling.js b/node_modules/ethers/lib.esm/providers/subscriber-polling.js
index f40de11..0eb4fad 100644
--- a/node_modules/ethers/lib.esm/providers/subscriber-polling.js
+++ b/node_modules/ethers/lib.esm/providers/subscriber-polling.js
@@ -228,6 +228,7 @@ export class PollingEventSubscriber {
         }
         const filter = copy(this.#filter);
         filter.fromBlock = this.#blockNumber + 1;
+        filter.fromBlock = this.#blockNumber; // Updated to fix a bug in polling scans for the same filtered events across consecutive blocks (ie. during automated tests).
         filter.toBlock = blockNumber;
         const logs = await this.#provider.getLogs(filter);
         // Undo debounce state
